src/main/java/org/icgc_argo/workflowgraphnode/rabbitmq/NodeConfiguration.java

```diff
@@ -161,7 +159,8 @@ public Disposable inputToRunning() {
         // flatMap needs a function that returns a Publisher that it then
         // resolves async by subscribing to it (ex. mono)
         //        .flatMap(node.gqlQuery())
-        .map(tx -> tx.map(toMap(tx.get()))) // TODO: temp until we move RDPC client
+        // TODO: execute gql query with GraphEvent and map tx to GQLResponse
+        .map(tx -> tx.map(Map.<String, Object>of("data", "value")))
```

Comment:
It was confusing me that the GraphEvent transaction (formely string) was going into the activation function...

Reply:
yes currently the GQL query is absent, will remedy this afternoon